25-hosted-ce-setup.sh

```diff
@@ -21,11 +21,10 @@ popd
 users=$(cat /etc/grid-security/grid-mapfile /etc/grid-security/voms-mapfile | \
             awk '/^"[^"]+" +[a-zA-Z0-9\-\._]+$/ {print $NF}' | \
             sort -u | \
-            tr '\n' ',')
+            tr '\n' ' ')
 [[ -n $users ]] || { echo >&2 "No users found in /etc/grid-security/grid-mapfile or /etc/grid-security/voms-mapfile"; exit 1; }
```

Comment:
Won't this always be true if there's a trailing space/comma in $users?

Reply:
If there are no users, the pipeline will result in the empty string:

```
[root@osg-hosted-ce-blin-test-ce-85f745c7cd-pm6rm ~]# cat grid-mapfile voms-mapfile
[root@osg-hosted-ce-blin-test-ce-85f745c7cd-pm6rm ~]# users=$(cat grid-mapfile voms-mapfile | \
>             awk '/^"[^"]+" +[a-zA-Z0-9\-\._]+$/ {print $NF}' | \
>             sort -u | \
>             tr '\n' ' ')
[root@osg-hosted-ce-blin-test-ce-85f745c7cd-pm6rm ~]# [[ -n $users ]] || { echo >&2 "No users found in /etc/grid-security/grid-mapfile or /etc/grid-security/voms-mapfile"; } 
No users found in /etc/grid-security/grid-mapfile or /etc/grid-security/voms-mapfile
```